public/sass/_variables.dark.generated.scss

```diff
@@ -112,8 +112,8 @@ $text-shadow-faint: 1px 1px 4px rgb(45, 45, 45);
 $textShadow: none;
 
 // gradients
-$brand-gradient-horizontal: linear-gradient(to right, #f05a28 30%, #fbca0a 99%);
-$brand-gradient-vertical: linear-gradient(#f05a28 30%, #fbca0a 99%);
+$brand-gradient-horizontal:  linear-gradient(270deg, #F55F3E 0%, #FF8833 100%);;
+$brand-gradient-vertical: linear-gradient(0.01deg, #F55F3E 0.01%, #FF8833 99.99%);;
```

Comment:
Don't know how these are generated, but any reason there are two semi colons at the end of each of these lines? 🤔

Reply:
that's a good question! I've made a change, but will need to check it's okay!!!